text/0000-fix-fees.md

```diff
@@ -0,0 +1,113 @@
+- Title: fix-fees
+- Authors: [John Tromp](mailto:john.tromp@gmail.com)
+- Start date: August 23, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000) 
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+Change Grin's minimum relay fees to be weight proportional and make output creation cost about a Grin-cent.
+Use least significant bits in fee to specify desired tx priority.
+
+# Motivation
+[motivation]: #motivation
+
+The current fee requirements suffer from being somewhat arbitrary, and not miner incentive compatible.
+They are not even linear; the fees required for the aggregate of two transactions is not necessarily equal to the sum
+of required fees.
+The current (and foreseeable) low price of Grin makes spamming the UTXO set rather cheaper than desired.
+Fee overpaying for higher priority to be included in full blocks fails when aggregated with minimal fee transactions.
+
+# Community-level explanation
```

Comment:
Just a comment that this section currently is a single paragraph. You may want to add extra line breaks where you want to improve readability and separate sections.